app/views/layouts/application_body_api_docs.html.erb

```diff
@@ -1,9 +1,9 @@
-<%# Copyright 2011-2013 Rice University. Licensed under the Affero General Public 
+<%# Copyright 2011-2013 Rice University. Licensed under the Affero General Public
     License version 3 or later.  See the COPYRIGHT file for details. %>
 
 <% content_for :javascript do %>
   <script type='text/javascript'>
-    $(document).ready(function() {
+    $(document).on('turbolinks:load', function() {
```

Comment:
What is this turbolinks thing?

Reply:
It's a gem used in new Rails apps since Rails 4 or 5 that intercepts clicks on links/buttons and replaces them with JS that fetches the new page, replaces the content, updates the url etc. Single page app without having to do it in JS (and maybe with more calls to the backend than an SPA would normally have).

Downsides are that it does not properly support `remote: true (local: false)` forms (this may change for Rails 6) and you have to replace all the `$(document).ready` stuff with a listener to this load event, because ready does not fire on navigation.

I added an attribute to the webview layout that should disable turbolinks there, since we don't want it interfering with Nathan's react stuff. So it should only do anything in the home page and admin pages.